### PR TITLE
Fix paths to Symfony config file

### PR DESCRIPTION
**Pull request by dunglas**
This PR also improves a bit the Markdown.

### CODE REVIEW

**Comment by dunglas** [comment]
If Symfony Flex is used and contrib recipes are authorized, then there is nothing to do at all.

**Comment by ramsey** [comment]
> If Symfony Flex is used and contrib recipes are authorized, then there is nothing to do at all.

I don't use Symfony. Can you explain what you mean by this?

File: README.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ the following in your bootstrap:
 In Symfony:
 
 ``` yaml
-# app/config/config.yml
+# app/config/packages/doctrine.yaml
```
Comment (maxhelias) [inline README.md]: `app` can also be remove 😉

**Comment by maxhelias** [comment]
@ramsey When you install the package with Symfony Flex, Flex automatically configures the projects with this  recipe : 
https://github.com/symfony/recipes-contrib/blob/master/ramsey/uuid-doctrine/1.3/config/packages/ramsey_uuid_doctrine.yaml

**Comment by ramsey** [comment]
Thank you for contributing! 🎉

**Pull request merged**